src/Nethermind/Nethermind.Core/Extensions/Bytes.cs

```diff
@@ -151,6 +151,34 @@ public static void SetBit(this ref byte b, int bitNumber)
             b |= (byte)mask;
         }
 
+        private unsafe static int GetAlignmentOffset(this byte[] array, uint alignment)
```

Comment:
I am not convinced that this is best place for this methods or should it be extension methods.
I think they should live in some `PinnedExtensions` class or something, so it won't be accidentally called on non-pinned objects.

Reply:
Moved to VirtualMachine as only place these methods are used